packages/aws-cdk-lib/aws-lambda/lib/function.ts

```diff
@@ -832,6 +841,7 @@ export class Function extends FunctionBase {
       codeSigningConfigArn: props.codeSigningConfig?.codeSigningConfigArn,
       architectures: this._architecture ? [this._architecture.name] : undefined,
       runtimeManagementConfig: props.runtimeManagementMode?.runtimeManagementConfig,
+      snapStart: this.configureSnapStart(props),
```

Comment:
Given my below comment, we could format this as follows:

```ts
snapStart: props.snapStart ? snapStart._render(props) : undefined,
```

Reply:
Thanks, wil check this out